js/indicators/indicators.src.js

```diff
@@ -4,12 +4,39 @@ import '../parts/Utilities.js';
 
 var pick = H.pick,
     each = H.each,
+    extend = H.extend,
+    wrap = H.wrap,
     error = H.error,
     Series = H.Series,
     isArray = H.isArray,
     addEvent = H.addEvent,
     seriesType = H.seriesType;
 
+/*
+ * Support for OHLC data with line type of series.
+ */
+wrap(H.Series.prototype, 'init', function (proceed, chart, options) {
```

Comment:
Can we add `init` event in the core?

Reply:
No reference to current series

Reply:
I mean add in the core `fireEvent(series, 'init')` then use `addEvent(H.Series, 'init', function () { .. });` instead of `wrap()`.